app/scripts/modules/kubernetes/loadBalancer/details/details.controller.js

```diff
@@ -23,22 +24,26 @@ module.exports = angular.module('spinnaker.loadBalancer.kubernetes.details.contr
       loading: true
     };
 
-    function extractLoadBalancer() {
+    let extractLoadBalancer = () => {
       return application.loadBalancers.ready()
         .then(() => {
-          $scope.loadBalancer = application.loadBalancers.data.filter(function (test) {
+          $scope.loadBalancer = application.loadBalancers.data.find((test) => {
             return test.name === loadBalancer.name &&
               (test.namespace === loadBalancer.region || test.namespace === loadBalancer.namespace) &&
               test.account === loadBalancer.accountId;
-          })[0];
+          });
 
           if ($scope.loadBalancer) {
+            this.ingressProtocol = 'http:';
+            if (_.get($scope.loadBalancer, 'service.spec.ports', []).some(p => p.protocol === 'HTTPS')) {
```

Comment:
There is unfortunately no [`HTTPS` protocol](https://godoc.org/k8s.io/kubernetes/pkg/api#Protocol). Could you instead check if `p.port == 443`?